src/ResourceManager/EventHub/Commands.EventHub/Cmdlets/DRConfig/RemoveAzureEventHubGeoDRConfiguration.cs

```diff
@@ -0,0 +1,106 @@
+        [ValidateNotNullOrEmpty]
+        public string Namespace { get; set; }
+
+        [Parameter(Mandatory = true, ParameterSetName = GeoDRBreakPairFailOverParameterSet, ValueFromPipelineByPropertyName = true, Position = 2, HelpMessage = "Alias (GeoDR)")]
+        [ValidateNotNullOrEmpty]
+        public string Name { get; set; }
+
+        [Parameter(Mandatory = true, ParameterSetName = GeoDRInputObjectParameterSet, ValueFromPipeline = true, Position = 0, HelpMessage = "Eventhub GeoDR Configuration Object")]
+        [ValidateNotNullOrEmpty]
+        public PSEventHubDRConfigurationAttributes InputObject { get; set; }
+
+        [Parameter(Mandatory = true, ParameterSetName = GeoDRConfigResourceIdParameterSet, ValueFromPipelineByPropertyName = true, Position = 0, HelpMessage = "GeoDRConfiguration Resource Id")]
+        [ValidateNotNullOrEmpty]
+        public string ResourceId { get; set; }
+
+        [Parameter(Mandatory = false)]
+        public SwitchParameter PassThru { get; set; }
+
+        public override void ExecuteCmdlet()
+        {
+            if (ParameterSetName == GeoDRInputObjectParameterSet)
+            {
+                ResourceIdentifier getParamGeoDR = GetResourceDetailsFromId(InputObject.Id);
+
+                if (getParamGeoDR.ResourceName != null && getParamGeoDR.ParentResource != null && getParamGeoDR.ResourceName != null)
+                {
+                    Client.DeleteEventHubDRConfiguration(getParamGeoDR.ResourceGroupName, getParamGeoDR.ParentResource, getParamGeoDR.ResourceName);
+                    if (PassThru)
+                    {
+                        WriteObject(true);
+                    }
+                    else
+                    {
+                        WriteObject(false);
+                    }
+                }
+            }
+
+            if (ParameterSetName == GeoDRConfigResourceIdParameterSet)
+            {
+                ResourceIdentifier getParamGeoDR = GetResourceDetailsFromId(ResourceId);
+
+                if (getParamGeoDR.ResourceName != null && getParamGeoDR.ParentResource != null && getParamGeoDR.ResourceName != null)
+                {
+                    Client.DeleteEventHubDRConfiguration(getParamGeoDR.ResourceGroupName, getParamGeoDR.ParentResource, getParamGeoDR.ResourceName);
+                    if (PassThru)
+                    {
+                        WriteObject(true);
+                    }
+                    else
+                    {
+                        WriteObject(false);
+                    }
+                }
+            }
+
+            if (ParameterSetName == GeoDRBreakPairFailOverParameterSet)
+            {
+                Client.DeleteEventHubDRConfiguration(ResourceGroupName, Namespace, Name);
+                if (PassThru)
+                {
+                    WriteObject(true);
+                }
+                else
```

Comment:
Same comment about else (probably doesn't need the else block in this parameter set)

Reply:
parameter set name changed to GeoDRParameterSet